### ISSUE TITLE
[Temporarily] Replace search with Google site search (or "Search is broken")

### ISSUE THREAD

**Issue by plusjeff** [department-of-veterans-affairs/vets-website#1960]
There's a few problems with the current search:

- It's not re-indexing (and the re-indexing script is broken with no maintainer), so no new sections have search results
- Rumor there's no technical person behind DigitalGov Search anymore (just a rumor, but a concerning one)
- Search result descriptions are ugly (try searching `employment` and looking at page descriptions)
- The search refinements (along the top) are not likely useful to users

A quick solution to make this better would be to simply send a search query to Google for just vets.gov (using `site:vets.gov` in the query).

Here's a super barebones implementation (no auto-suggest), but functional no-less:

```html
<html>

<body>
    <form accept-charset="UTF-8" action="https://www.google.com/search" method="get">
        <input id="search_query" name="q" type="hidden" />
        <input id="vetsgov_query" name="vetsgov_query" type="text" />
        <input type="submit" value="Search" onclick="makeSearchQuery()" />
    </form>
</body>

<script type="text/javascript">
    function makeSearchQuery() {
        vetsgovQuery = document.getElementById("vetsgov_query");
        searchQuery = document.getElementById("search_query");
        searchQuery.value = vetsgovQuery.value + " site:vets.gov";
    }
</script>

</html>
```

To complete this, search would have to be replaced here in vets-website, but also in applications that share the header.

Good things about this:

- Great search results that are good/accurate for users
- No maintenance for indexing the site

Bad things:

- Users leave vets.gov's look and feel (although, all the search results will send them back)
- Ads are possible (in my tests, there were none, but that doesn't prevent it from happening in the future)

**Comment by ayaleloehr** [comment]
FWIW, I believe all applications are using the va_common gem for their header, so the most effective way to get this working for applications would be to update that gem and have each application update to the new version. 

As to the first con you mention, I don't feel that the search page we have today necessarily has a "vets.gov" look and feel anyways (it has a different header and no footer, for example). And the URL structure of the search results today isn't even "vets.gov" but instead is "search.usa.gov", so I don't feel that this is actually a con overall.  We also don't have auto-suggestions currently, so that wouldn't be a change either. 

The biggest con to me is that we won't have logging on what the most common search queries are or have a way to make sure users are able to get to their common questions in one click, which I know @MarinaMartin cares greatly about.

No matter what, I think we need some solution to search implemented and in production this week, as we have launched Careers and Employment and need to get that indexed ASAP.  It's going to be a decision as to how much time we can find to focus on this that decides whether this fix (which I think is awesome you researched and super appreciate you figuring out @plusjeff!) or getting DigitalGov Search working better again is the right approach.  @plusjeff let's chat Monday morning about what priorities are this week to figure that out?  It's likely going to be you or I doing this, but also cc @alexose as I know he starts soon and might have opinions here.

**Comment by MarinaMartin** [comment]
There has to be some way to log the searches through our google analytics account? (But I agree that for now, let's do this, then we can figure out the search term tracking and probably ideally showing the results within a header and footer)

**Comment by ayaleloehr** [comment]
Closed via commit 4addaa245dffb7d34a1a10cf10e8777840e302fb
